omnibus/.rubocop.yml

```diff
@@ -5,7 +5,6 @@ AllCops:
   DisplayCopNames: true
   Exclude:
     - "../*/vendor/**/*"
-    - "../*/bin/**/*"
```

Comment:
[Run from this change](https://github.com/dependabot/dependabot-core/actions/runs/5836540290/job/15830340443?pr=7798): 846 files inspected
[Run from earlier today](https://github.com/github/dependabot-api/pull/4010): 843 files inspected

I had expected more than three new files to be inspected, but it tracks with 👇 (`/updater/bin/run` is a Bash script)

```plaintext
 ~/github/dependabot-core/omnibus $ ll ../*/bin
../common/bin:
total 8
drwxr-xr-x   3 me  staff    96B Nov 15  2021 .
drwxr-xr-x  11 me  staff   352B Jul  7 13:23 ..
-rwxr-xr-x   1 me  staff   297B Nov 15  2021 git-credential-store-immutable

../updater/bin:
total 24
drwxr-xr-x   5 me  staff   160B Apr 19 15:41 .
drwxr-xr-x  13 me  staff   416B Jul 13 11:20 ..
-rw-r--r--   1 me  staff   583B Apr 19 15:41 fetch_files.rb
-rwxr-xr-x   1 me  staff   370B Sep  6  2022 run
-rw-r--r--   1 me  staff   584B Apr 19 15:41 update_files.rb
```